Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7.4-slim
+
+WORKDIR /srv/app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache -r requirements.txt \
+    && rm -rf /tmp/*
+
+COPY . .
+
+RUN pip install -e .
```

Comment:
what does this do? Installs the current package globally into the system (the container in this case)?

Reply:
installs local files as a python package according to `setup.py`. It's useful for when you need to make changes otherwise you would have to rebuild your image after making changes.